js/app.js

```diff
@@ -11,6 +11,10 @@ const secondCallTimer = document.querySelector('#secondCallNotification #call-ti
 const profileDropDown = document.getElementById("myDropdown");
 const profileOnline = document.querySelector(".dropbtn #availability");
 
+let service_app_token = 'MzA3MWQ4NGEtNDA5Ni00MDIwLThkOTctOGZlMmI1N2M1ODk5ZTA0NjAyZDYtNTk5_PF84_ef70fae6-b079-45e6-901e-0a1ba9856721'; // 6100 queue
+// let service_app_token = 'YTgyZjkxNTMtM2IzOC00NjYwLTlhMmQtMjk2Y2EzMjUyY2Y2MGFkN2ZkNjAtZTc5_P0A1_b657fcf3-6b06-46a4-ab0e-f23fceb7a447'; //5200 queue
+// const refresh_token = 'ZDQzZDE3MjgtYThmNS00NzkwLThhZGMtYjdmYzMxNjAyOTQwZmM1ZGY4NDgtOTU4_PF84_ef70fae6-b079-45e6-901e-0a1ba9856721';
```

Comment:
Please remove sensitive token info and only leave CONSTANTS which devs have to fill up. Similar to our instructions in the beta

Reply:
done